Assets/MixedRealityToolkit/UX/Scripts/Tooltip/ToolTipSpawner.cs

```diff
@@ -0,0 +1,263 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System.Collections;
+using UnityEngine;
+using MixedRealityToolkit.InputModule;
+using MixedRealityToolkit.InputModule.InputHandlers;
+using MixedRealityToolkit.InputModule.EventData;
+
+
+namespace MixedRealityToolkit.UX.ToolTips
+{
+    /// <summary>
+    /// Add to any Object to spawn ToolTips on tap or on focus, according to preference
+    /// Applies its follow settings to the spawned ToolTip's ToolTipConnector component
+    /// </summary>
+    public class ToolTipSpawner : MonoBehaviour , IInputHandler, IPointerSpecificFocusable
+    {
+        public enum VanishBehaviorEnum
```

Comment:
Same comment as with Tooltip Connector, either generalise these for all UX as separate public scripts or make them private. As well as removing enum suffix